ui_framework/src/components/form/form_range/_form_range.scss

```diff
@@ -0,0 +1,83 @@
+// The following code is inspired by...
```

Comment:
I modified this quite a bit. It's small enough I didn't want to install it as a dependency, because the overwrites would have been just as large as the original source. Let me know if this is the proper way to credit stuff in our code.